src/normalize/glossary.ts

```diff
@@ -174,6 +167,13 @@ export default function normalizeGlossary(editor: Editor, entry: NodeEntry): boo
         }
     }
 
+    // We look for Term in DefinitionTerm, unwrap it if found
+    if (Term.isTerm(node) || Title.isTitle(node) || Paragraph.isParagraph(node)) {
```

Comment:
You don't need to check for `Title` and `Paragraph`. These are block nodes, and so cannot be children of `DefinitionTerm`.